ios/RNSScreenStackAnimator.mm

```diff
@@ -22,7 +23,7 @@ - (instancetype)initWithOperation:(UINavigationControllerOperation)operation
 {
   if (self = [super init]) {
     _operation = operation;
-    _transitionDuration = 0.35; // default duration in seconds
+    _transitionDuration = 0.5; // default duration in seconds
```

Comment:
Also does it not differ from one iOS version to another? Value of 0.35s is around for longer that I'm in this project 😅 but I'm **almost** sure that when it was introduced it was meant to resemble native animation as close as possible, hence my impression that this differs between various iOS versions. It would be nice to have some kind of reference, because changing it here to e.g. 0.5s might look "more natively" on iOS 18 but "less natively" on iOS 16 - this is my main objection right now.